data/anurag-rajawat.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Anurag Rajawat",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source is ❤️ | Gooogle Summer of Code 2022 contributor",
+  "avatar": "https://github.com/anurag-rajawat.png",
+  "tags": ["Open Source", "Kubernetes", "Java", "DevOps"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/anuragrajawat22" },
+    { "platform": "github", "url": "https://github.com/anurag-rajawat" },
+    {
+      "platform": "linkedIn",
+      "url": "https://www.linkedin.com/in/anurag-rajawat22/"
+    }
+  ],
+  "links": [
+    {
+      "name": "GitHub: Let's Collaborate",
```

Comment:
```suggestion
      "name": "Let's Collaborate",
```

will look like this 👇
![image](https://user-images.githubusercontent.com/61644078/208914371-3c971868-92b4-44eb-8b41-29a0d14933cf.png)


Please ignore this suggestion if it was intended.

Reply:
I missed it. Thanks for pointing out.